src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,43 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type PropsForCard = Omit<Movie, 'imdbId'>;
```

Comment:
as I understood Misha's video, He gave advise don't make complicated type for props
https://youtu.be/LGqqEg1Z4FA?t=160